src/styles/images/index.md.njk

```diff
@@ -43,7 +43,7 @@ Do not:
 - include the name of the photographer or person who created the image
 - start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
 - repeat information from the page
-- include extra information not on the page
+- include extra information not in the image
```

Comment:
Suggestion: 
```suggestion
- include additional information not mentioned on the page
```

I think the aim of this line is to make sure all users can receive the same information regardless of whether they are using assistive technologies. Equally, the repetition of content to a user using assistive technology could be confusing and potentially perceived as an error.